ports/cpp/source/antlr4-c3/CodeCompletionCore.cpp

```diff
@@ -335,8 +335,8 @@ FollowSetsHolder CodeCompletionCore::determineFollowSets(antlr4::atn::ATNState *
 	
 	return {
 		.sets = sets,
+		.combined = combined,
 		.isExhaustive = isExhaustive,
-		.combined = combined
 	};
```

Comment:
I'm not familiar with that syntax. Is that C++20? Only know that from Swift (with a different meaning).

Reply:
As I understand, it is called [Aggregate Initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization) and exists to simplify object creation.

So it lets you initialize plain structs this way:
```
struct Point {
  std::size_t x;
  std::size_t y;
}

Point point = {.x = 10, .y = 10};
```

It is since `C++20`.